Extensions/Video/videoruntimeobject-pixi-renderer.js

```diff
@@ -22,8 +22,11 @@ gdjs.VideoRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
   }
   this._textureWasValid = false; // Will be set to true when video texture is loaded.
 
-  if (!this._pixiObject.texture.baseTexture.source.paused) {
-    this._pixiObject.texture.baseTexture.source.pause();
+  if (
+    this._textureWasValid &&
```

Comment:
This will always be false :) So not sure it's the right place to pause the video.

Reply:
> The prevent seem working but it's not, chrome stop automatically the play because user didn't interact with the document.

Yeah I've seen this. Update Pixi won't help. For scene editor is fine because we don't want auto play or play at all.
For games it's more a problem. We'll have to make it clear to users that if they want to play a video, they have to get a click from the user at the beginning of the game.

Reply:
When all versions of pixi inside GD will be same (v4.8.6 require), the problems deaseapear because autoplay will be useable.